hermes/utils.py

```diff
@@ -25,78 +25,6 @@
 logger = logging.getLogger(__name__)
 
 
-TNS_TYPES = [
```

Comment:
Do we not need these anymore? I didn't see this reproduced anywhere.

Reply:
We get TNS options from their API - this was just an unused holdover from before we had a connection to TNS to get the options from them.